src/main/java/duke/TaskStorage.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import java.io.*;
```

Comment:
I think it would be better to list out the individual dependencies you wish to import instead of using a wildcard